Circle.cpp

```diff
@@ -6,16 +6,14 @@
 #include <iostream>
 #include <math.h>
 
-#include "Shape.cpp"
-#include "Area.cpp"
-#include "Volume.cpp"
-#include "Input.cpp"
-
+#include "ShapeManager.cpp"
 
 using namespace std;
-class Circle : public Shape
-{
+class Circle : public ShapeManager  
+//Responsibility: This class is responsible to provide implementation for draw(), inputDimensions, calculateArea(), and calculateVolume().
```

Comment:
You cannot use function names as the class's responsiblity. This is like defining a meaning of a word using the same word.

You need to focus on what drives the class's existence / creation or in other words what purpose the class is to fulfil.

The next step comes after: i.e. the purpose of the class should be single.

Once you've identified how the class violates that single responsibility principle, you need to state how it does so.